features/step_definitions/run_steps.rb

```diff
@@ -63,8 +63,9 @@
 end
 
 
-Then(/^I see "(.+?)"$/) do |output|
-  expect(@last_run_result.out).to eql "#{output}\n"
+Then(/^I(?: do|( don't))? see "(.*)"$/) do |verb, string|
+  method = verb == "don't" ? :not_to : :to
+  expect(@last_run_result.out).public_send(method, include(string))
 end
```

Comment:
Ok, I kept `include` in the "don't see" version, as I think it still makes sense there